SPECS/nbdkit/nbdkit.spec

```diff
@@ -1193,6 +1195,10 @@ export LIBGUESTFS_TRACE=1
 
 
 %changelog
+* Wed Jun 18 2025 Durga Jagadeesh Palli <v-dpalli@microsoft.com> - 1.35.3-4
```

Comment:
Can we please once verify that the same tests are failing before and after applying the patch and that we have not introduced any new test regression.